eagle-core/eagle-alert-parent/eagle-alert-service/src/main/java/org/apache/eagle/service/alert/resolver/AttributeResolverFactory.java

```diff
@@ -43,8 +43,6 @@ public static AttributeResolvable getAttributeResolver(String fieldResolverName,
                 fieldResolvableCache.put(fieldResolverName, instance);
             } catch (ClassNotFoundException e) {
                 throw new AttributeResolveException("Attribute Resolver in type of " + fieldResolverName + " is not found", e);
-            } catch (InstantiationException | IllegalAccessException e) {
```

Comment:
instead of removing this, it can be used to describe the error, IDE is suggesting to collapse the catch block may be because it has identical error msg (throw new AttributeResolveException(e)).